### PR TITLE
✨ gRPC conversion

### PR DESCRIPTION
**Pull request by dmihalcik-virtru**
Idea: write service in gRPC, use grpc-gateway to expose alternative, swagger API. Unfortunately, this means it must be openapi 2 still. 

https://github.com/grpc-ecosystem/grpc-gateway/issues/441

Some notes:

1. `/kas_public_key` can either return a `string` or a message. AFAICT this means I have to implement it using [an HTTP Body message](https://grpc-ecosystem.github.io/grpc-gateway/docs/mapping/httpbody_messages/) to emulate existing behavior. As such, I've created two gRPC services, one which returns the newer, structured type (the preferred on to call for gRPC clients) and another, legacy compatible version that is not intended to be used by gRPC clients. We'll have to switch to HttpBody to implement `kid` and `jwk` support in the legacy endpoint
2. I'm checking in the generated code golang code. I don't think this is required, since we aren't a library, but it may enhance code reviews and browsing.
3. I'm also checking in the swagger.yaml, only for review. It is currently unused in any way, although it could be used to generate client libraries.

### CODE REVIEW

File: client/main.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/*
```
Comment (pflynn-virtru) [inline client/main.go:1]: I would move this to either `examples/` or `tests/`.  For non-tests/examples, put it under `cmd/`

File: pkg/access/service.proto [hunk 2]
```diff
@@ -0,0 +1,94 @@
+syntax = "proto3";
```
Comment (pflynn-virtru) [inline pkg/access/service.proto:1]: putting photos here, I believe, is a recommendation from Google, but in the pic repo, we are placing protos in `proto/` and `/gen`.  We should do a comparative analysis (a meeting)  on which one is better for us.

File: client/main.go (hunk 1 above)
Comment (dmihalcik-virtru) [reply client/main.go]: Moved to examples/client/main.go

File: client/main.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	"google.golang.org/grpc/credentials/insecure"
+	_ "google.golang.org/grpc/health"
+)
+
+func callVersion(c access.AccessServiceClient) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	r, err := c.Info(ctx, &access.InfoRequest{})
+	if err != nil {
+		fmt.Println("AppInfo: _, ", err)
```
[the remark this answers is not in the document]
Comment (dmihalcik-virtru) [reply client/main.go]: Okay

File: pkg/access/service.proto (hunk 2 above)
Comment (dmihalcik-virtru) [reply pkg/access/service.proto:1]: Yeah I like them in the same folder/package (not surprising I suspect). I suggest we leave them as is for now and move them if it becomes an issue.

**Review by pflynn-virtru — APPROVED** [review]

**Pull request merged**